RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.37.0 (2019-08-15)
+==========================
+
+Language
+--------
+- [`#[must_use]` will now warn if the type is in a tuple and unused.][61100]
+- [`#[must_use]` will now warn if the type is in a `Box` and unused.][62228]
+- [`#[must_use]` will now warn if the type is in a array and unused.][62235]
+- [You can now use the `cfg` and `cfg_attr` attributes on
+  generic parameters.][61547]
+- [You can now use enum variants through type alias.][61682] e.g. You can
+  write the following.
+  ```rust
+  type MyOption = Option<u8>;
+
+  fn increment_or_zero(x: MyOption) -> u8 {
+      match x {
+          MyOption::Some(y) => y + 1,
+          MyOption::None => 0,
+      }
+  }
+  ```
+- [You can now use `_` as a identifier for consts.][61347] e.g. You can write
+  `const _: u32 = 5;`.
+- [You can now use `#[repr(align)]` on enums.][61229]
+- [The  `?`/_"Kleene"_ macro operator is now available in the
```

Comment:
I feel like based on a quick skim of wikipedia the kleene operator is specifically `*`, not `?` -- but maybe I'm wrong? I haven't done enough research to be certain.

cc @Centril

Reply:
I would just call it "at most one". Otherwise I think kleene is right.